lib/creator/transformations/devtool/devtool.js

```diff
@@ -5,8 +5,11 @@ module.exports = function(j, ast, yeomanConfig) {
 	const webpackProperties = yeomanConfig.webpackOptions;
 
 	function createDevToolProperty(p) {
-		const propValue = j.literal(webpackProperties['devtool']);
-		transformUtils.checkIfExistsAndAddValue(j, p, 'devtools', propValue);
```

Comment:
Oh right. Totally forgot. It's literal value. We need to change this based on if it's an actual literal or if it should be treated as an identifier